subsys/nrf_security/src/core/Kconfig

```diff
@@ -19,4 +19,18 @@ config PSA_CORE_OBERON
 	select PSA_WANT_AES_KEY_SIZE_192
 	select PSA_WANT_AES_KEY_SIZE_256
 
+config PSA_CORE_LITE
+	bool "PSA core created for tiny footprint"
+	depends on SOC_SERIES_NRF54LX
+	help
+	  The PSA core with tiny footprint is created to be used e.g. for bootloader
+	  or similar use-cases where there is severe size-restrictions. This PSA core
+	  will only provide limited algorithm support and only a limited amount of features
+	  from the PSA crypto key management APIs. Note that volatile key-store is not
+	  possible to use.
```

Comment:
Please also note that that PSA core lite is not thread safe since it uses global buffers.

Reply:
Good point! Fixed